File: server/fix.js
import mongoose from 'mongoose';
import fs from 'fs';

(async () => {
  await mongoose.connect('mongodb://127.0.0.1:27017/skillbridge');
  const text = fs.readFileSync('..\\ml-service\\resume_text.txt', 'utf-8');
  console.log('Read text length:', text.length);
  await mongoose.connection.db.collection('users').updateMany({}, { $set: { resumeText: text } });
  await mongoose.connection.db.collection('analysissnapshots').deleteMany({});
  console.log('Updated users and cleared snapshots!');
  process.exit(0);
})();

File: ml-service/fix.py
import pymongo
from services.text_extractor import extract_text_from_file

client = pymongo.MongoClient("mongodb://127.0.0.1:27017/")
db = client["skillbridge"]

pdf_path = r"C:\Users\Abhishek_Anil\.gemini\antigravity-ide\brain\4bf90252-550d-444f-a671-c74a608ca032\.user_uploaded\media_1788018487602.pdf"
text = extract_text_from_file(pdf_path)

print("Extracted length:", len(text))

db.users.update_many({}, {"$set": {"resumeText": text}})
db.analysissnapshots.delete_many({})

print("Updated users and cleared snapshots!")
